bin/postgres-ha/post-existing-init.sql

```diff
@@ -0,0 +1,2 @@
+CREATE USER ${PATRONI_SUPERUSER_USERNAME} WITH SUPERUSER ENCRYPTED PASSWORD '${PATRONI_SUPERUSER_PASSWORD}';
```

Comment:
@andrewlecuyer The `ENCRYPTED` keyword does not exist in newer versions of PostgreSQL. I can't remember when it was deprecrated, but that keyword can be removed.

Reply:
The `ENCRYPTED` parameter has been removed.